packages/next/build/webpack/config/index.ts

```diff
@@ -0,0 +1,40 @@
+import webpack from 'webpack'
+import { base } from './blocks/base'
+import { css } from './blocks/css'
+import { ConfigurationContext, pipe } from './utils'
+import { experimentData } from './blocks/experiment-data'
+
+export function build(
+  config: webpack.Configuration,
+  {
+    rootDirectory,
+    customAppFile,
+    isDevelopment,
+    isServer,
+    hasSupportCss,
+    hasSupportData,
+  }: {
+    rootDirectory: string
+    customAppFile: string | null
+    isDevelopment: boolean
+    isServer: boolean
+    hasSupportCss: boolean
+    hasSupportData: boolean
+  }
+) {
+  const ctx: ConfigurationContext = {
+    rootDirectory,
+    customAppFile,
+    isDevelopment,
+    isProduction: !isDevelopment,
+    isServer,
+    isClient: !isServer,
```

Comment:
```suggestion
```
Doesn't look like this is used, is this just to have?

Reply:
Just to have for the future, yeah.